osu.Game/Skinning/LegacySkinConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Skinning
+{
+    public class LegacySkinConfiguration : DefaultSkinConfiguration
+    {
+        public const decimal LATEST_VERSION = 2.5m;
+
+        /// <summary>
+        /// Legacy version of this skin.
```

Comment:
what does null mean?

Reply:
that no version was set in the decoder, that leads into returning a `null` in `GetConfig(Version)`, as the user is forced into putting a default value in case the version retrieved is null.

```csharp
skin.GetConfig<LegacySkinConfigurations, decimal>(LegacySkinConfigurations.Version)?.Value ?? 1;
```